lang/en/organisations.php

```diff
@@ -13,12 +13,14 @@
     ],
     'members'       => [
         'actions'       => [
-            'add'       => 'Add a member',
-            'submit'    => 'Add member',
+            'add'               => 'Add a member',
```

Comment:
`add` be removed if it's not used anywhere else

Reply:
Still used